sys-block/thin-provisioning-tools/thin-provisioning-tools-9999.ebuild

```diff
@@ -27,14 +170,21 @@ src_unpack() {
 		git-r3_src_unpack
 		cargo_live_src_unpack
 	else
-		default
+		cargo_src_unpack
 	fi
 }
 
-src_compile() {
-	emake V= STRIP=true
+src_configure() {
+	local myfeatures=( $(usex io-uring io_uring "") )
```

Comment:
```suggestion
	local myfeatures=( $(usev io-uring) )
```

Reply:
The feature name uses an underscore while the USE flag, common to six other packages, uses a hyphen.

Reply:
Indeed, and in fact we have a documented policy against using underscores in `USE` flag names outside of `USE_EXPAND`. See https://projects.gentoo.org/qa/policy-guide/use-flags.html#pg0803

Reply:
oh sorry yes, then use `$(usev io-uring io_uring)`